tests/apps/pubsub-bulk-subscriber/app.go

```diff
@@ -0,0 +1,577 @@
+/*
```

Comment:
In this folder, please add a file called `.cache-include` with this content:

```
../utils/*.go
```

See #5278

Reply:
https://github.com/dapr/dapr/pull/5285 addresses this one.